ai/Get_started_LiveAPI.py

```
@@ -0,0 +1,234 @@
+                tg.create_task(self.play_audio())
+
+                await asyncio.Event().wait()  # 무한 대기, Ctrl+c로 종료
+
+        except asyncio.CancelledError:
+            pass
+        except KeyboardInterrupt:
+        # Ctrl+C로 종료될 때 대화 기록 저장
+            with open("conversation_log.json", "w", encoding="utf-8") as f:
+                json.dump(self.conversation_log, f, ensure_ascii=False, indent=2)
+            print("대화 기록이 저장되었습니다.")
+        except ExceptionGroup as EG:
```

Ruff (0.11.9)

Undefined name `ExceptionGroup`. Consider specifying `requires-python = ">= 3.11"` or `tool.ruff.target-version = "py311"` in your `pyproject.toml` file.

(F821)